src/dataexplorer/dataexplorer-application/src/types/IHistory.ts

```diff
@@ -0,0 +1,5 @@
+import { History } from 'history';
+
+export interface IHistory {
+    history: History;
+}
```

Comment:
If it works to use History directly as suggested over, this file could be deleted